python/ray/data/context.py

```diff
@@ -534,6 +535,9 @@ class DataContext:
         default_factory=_issue_detectors_config_factory
     )
 
+    downstream_capacity_backpressure_ratio: float = float("inf")
+    downstream_capacity_backpressure_max_queued_bundles: int = sys.maxsize
```

Comment:
Let's by default set this to None to make clear they are unset and then where we use them we'll fallback to defaults (otherwise there's no way to tell whether user set these values or they are defaults)